server/src/main/java/org/elasticsearch/index/IndexVersions.java

```diff
@@ -161,6 +161,7 @@ private static Version parseUnchecked(String version) {
     public static final IndexVersion USE_LUCENE101_POSTINGS_FORMAT = def(9_021_0_00, Version.LUCENE_10_1_0);
     public static final IndexVersion UPGRADE_TO_LUCENE_10_2_0 = def(9_022_00_0, Version.LUCENE_10_2_0);
     public static final IndexVersion UPGRADE_TO_LUCENE_10_2_1 = def(9_023_00_0, Version.LUCENE_10_2_1);
+    public static final IndexVersion DEFAULT_OVERSAMPLE_VALUE_FOR_BBQ = def(9_024_0_00, Version.LUCENE_10_2_1);
```

Comment:
@elastic/es-core-infra apparently with 10.2.0 we messed up the index version format (supposed to be `M_NNN_S_PP`).

Are we OK switching it back?

//cc @thecoop

Reply:
Yes, this is fine, it's a source-only change (`_` doesnt affect the integer value)